src/Components/CommentCard.jsx

```diff
@@ -1,29 +1,50 @@
-import {useState} from 'react';
+import {useEffect, useState} from 'react';
+import { useContext } from 'react';
+import { UserNameContext } from '../Contexts/Users';
+import { deleteComment } from '../../api';
+import { getCommentsByArticle } from '../../api';
 
-const CommentCard = ({prop}) => {
+const CommentCard = ({prop, states}) => {
     const [votes, setVotes] = useState(prop.votes);
     const [isDisabled, setDisable] = useState(false)
     const incrementVote = () => {
         setVotes(votes+1)
         setDisable(true)
-
     }
+    const {userName, setUserName} = useContext(UserNameContext);
+    const [commentDisplay, setCommentDisplay] = useState(prop)
+  
+    
+    
+    
     const decrementVote = () => {
         if (votes>0)
         setVotes(votes-1)
     }
-    
-  
+    const handleDeleteBTN = (event) => {
+
+        event.preventDefault()
+        deleteComment(prop.comment_id).then(()=>{
+            return getCommentsByArticle(prop.article_id)
+        }).then(({comments}) => {
+            setCommentDisplay(comments)
+        });
```

Comment:
This is great, some `loading` state could be good to include, as a small little tag that shows up to show a user it's being processed, also a `catch` and some loading state would be great to show a user when it rejects